portfolio/src/main/webapp/style.css

```diff
@@ -86,14 +86,24 @@ h1 {
     padding-left: 10px;
     padding-right: 10px;
 }
-/* .subPar:hover {
-    background-color: lightcyan;
-} */
 
-#about-container {
-    background-color: lightcyan;
+.activeTab{
+    background-color: rgb(230, 87, 0);
+    color: antiquewhite;
+    text-shadow: rgb(108, 160, 115);
+    border-radius: 10px 10px 0px 0px;
 }
 
+#about-container { 
+    background-color: rgb(230, 87, 0);
```

Comment:
This is just a minor suggestion but I noticed that you use the color `rgb(230, 87, 0)` a lot. From some searching around, it seems like you can define variables in CSS like so: 
``` 
:root {
    --backgroundcolor: rgb(230, 87, 0);; 
}

html, body {
    background-color: var(--backgroundcolor); 
} 
```
Maybe this could improve clarity?